webrtc-stats.html

```diff
@@ -3361,6 +3362,17 @@ <h2>
                   RTCDtlsTransport's "transport".
                 </p>
               </dd>
+              <dt>
+                <dfn><code>iceUfrag</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMString</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Set to the current ufrag value of the underlying transport.
+                  This can be used to identify which ICE generation associated
+                  ICE candidates belong to.
```

Comment:
Edge case: While ICE is restarting, could the transport update it's ufrag, but there exist candidates from both old and new generation present, e.g. unless you've found a new working candidate pair (from the new generation) you're still relying on the old candidate pair (from the old generation) assuming it's connection is not lost? Or does ICE restart mean all connections die?

Ice candidates reference the transport they belong to, but not which generation. I think the ufrag though is something that is set on the transport.